openlibrary/templates/books/edit.html

```diff
@@ -93,6 +93,7 @@ <h1>$:title</h1>
             <div class="label">
                 <label for="author-$0">$_('Author')</label>
                 <span class="tip">$:_('You can search by author name (like <em>j k rowling</em>) or by Open Library ID (like <a href="/authors/OL23919A" target="_blank"><em>OL23919A</em></a>).')</span>
+                <br/><noscript style="color:red">Author editing requires javascript</noscript>
```

Comment:
This should be translatable - to do that you just need to wrap it in `$:_(` like above (that function is not obvious at all so you weren't to know :)

Reply:
Instead of using the style attribute here let's make it look nire like an error using the existing pattern at https://github.com/internetarchive/openlibrary/wiki/Design-Pattern-Library#flashmessageerror (it doesn't look great but the more usages we identify the easier it will be to improve our error styling across the site).

See static/css/components/flash-messages.less for the HTML structure
You'll need to import that file in static/css/page-user.less  and run `make css` to see the styles.

```
<noscript>
<div class="flash-messages"><p>Author editing requires javascript</p></div>
</noscript>
```

Reply:
@jdlrobson I have implemented the changes you requested, but I believe I may be missing something..please let me know what that is

Reply:
I think it actually needs to be a span to get the required styling.

<img width="761" alt="Screen Shot 2020-04-27 at 10 00 50 AM" src="https://user-images.githubusercontent.com/148752/80399402-fb784f00-886d-11ea-975d-659744fd74ef.png">

Reply:
Hey @jdlrobson  I'm sorry for the delay..haven't been able to invest any time for the past week...
I've completed the changes and the internationalized flash message is showing as requested. But Instead of the error symbol, I'm getting a tick mark. 

![whatiswrong](https://user-images.githubusercontent.com/55073596/81256171-515aad00-904d-11ea-9967-5b70058e11ba.png)

Reply:
Great job!  If you look at 
https://github.com/internetarchive/openlibrary/blob/0818d6bc5ff1d4f87cac8e57a2c82c3be3b6cc33/static/css/components/flash-messages.less#L20 to switch out the tick for the cross the span itself must be inside an error element!

Reply:
Sorry, I couldn't comprehend.
here's what I understood, which doesn't work (obviously):

```
 .error {
    span {
```
Thanks a bunch